src/main/webapp/PlaceGuideOnList.js

```diff
@@ -0,0 +1,41 @@
+class PlaceGuideOnList {
+  constructor(placeGuideId, place, name, creator, description, audioKey, audioLength, imageKey, createdByCurrentUser) {
+    const placeGuideDiv = document.createElement("div");
+    placeGuideDiv.setAttribute('id', placeGuideId);
+    
+    placeGuideDiv.addEventListener('click', function() {
+      PlaceGuideManager.highlightPlaceGuide(placeGuideId);
+    });
```

Comment:
Please move this to a separate function, even if its short. You can use a naming pattern such as "highlightOnInfoboxClick", to make the code more readable than in case of using a listener directly in the constructor.

Reply:
created a function called `highlightOnInfoboxClick`